### ISSUE TITLE
Slippage option merged in #369 errors on BigNumber conversion when entering a dot first

### ISSUE THREAD

**Issue by j-h-scheufen** [Uniswap/uniswap-frontend#377]
**Bug Description**
With the intention of skipping the zero when entering a custom slippage value like `0.05` and starting with the dot `.` instead of `0` on beta.uniswap.exchange (see #369) the browser errors with

```
Error: underflow (operation="setValue", fault="underflow", value=null, outputValue=null, version=4.0.33)
    at Object.a [as throwError] (ethers.min.js:579)
    at new v (ethers.min.js:7658)
    at Object.y [as bigNumberify] (ethers.min.js:7681)
    at on (index.js:242)
    at Xo (react-dom.production.min.js:3274)
    at Pi (react-dom.production.min.js:3745)
    at Ni (react-dom.production.min.js:3957)
    at Ka (react-dom.production.min.js:5514)
    at Ga (react-dom.production.min.js:5536)
    at Ms (react-dom.production.min.js:5958)
da @ react-dom.production.min.js:4408
r.callback @ react-dom.production.min.js:4773
oa @ react-dom.production.min.js:4271
na @ react-dom.production.min.js:4259
Wa @ react-dom.production.min.js:4999
Va @ react-dom.production.min.js:5123
(anonymous) @ react-dom.production.min.js:5975
t.unstable_runWithPriority @ scheduler.production.min.js:274
js @ react-dom.production.min.js:5974
Ms @ react-dom.production.min.js:5958
Ps @ react-dom.production.min.js:5925
Es @ react-dom.production.min.js:5860
(anonymous) @ react-dom.production.min.js:5061
l @ scheduler.production.min.js:49
d @ scheduler.production.min.js:111
B.port1.onmessage @ scheduler.production.min.js:219
ethers.min.js:579 Uncaught Error: underflow (operation="setValue", fault="underflow", value=null, outputValue=null, version=4.0.33)
    at Object.a [as throwError] (ethers.min.js:579)
    at new v (ethers.min.js:7658)
    at Object.y [as bigNumberify] (ethers.min.js:7681)
    at on (index.js:242)
    at Xo (react-dom.production.min.js:3274)
    at Pi (react-dom.production.min.js:3745)
    at Ni (react-dom.production.min.js:3957)
    at Ka (react-dom.production.min.js:5514)
    at Ga (react-dom.production.min.js:5536)
    at Ms (react-dom.production.min.js:5958)
```

**Steps to Reproduce**
1. Enter trade pair and amount
2. Open Details and enter custom slippage value starting with a dot

**Expected Behavior**
Add the leading zero automatically

**Additional Context**
Tried on latest Chrome and Firefox

**Comment by NoahZinsmeister** [comment]
thanks @j-h-scheufen ! @ianlapham, mind taking a look at this?

**Comment by NoahZinsmeister** [comment]
fixed in #381
